crowbar_framework/app/models/api/node.rb

```diff
@@ -66,6 +70,7 @@ def post_upgrade
     end
 
     def join_and_chef
+      save_node_action("joining crowbar")
```

Comment:
I'd suggest something like: "upgrading configuration and re-joining the crowbar environment"